markdownlint.js

```diff
@@ -278,6 +296,9 @@ function lintAndPrint(stdin, files) {
       stdin
     };
   }
+  if (options.json) {
+    lintOptions.resultVersion = 3;
```

Comment:
Can't easily review this file from my phone, will do so later...

Reply:
Cool thanks.